plugin/geoip/city.go

```diff
@@ -6,27 +6,25 @@ import (
 
 	"github.com/coredns/coredns/plugin/metadata"
 
-	"github.com/oschwald/geoip2-golang"
+	"github.com/oschwald/geoip2-golang/v2"
 )
 
-const defaultLang = "en"
-
 func (g GeoIP) setCityMetadata(ctx context.Context, data *geoip2.City) {
 	// Set labels for city, country and continent names.
-	cityName := data.City.Names[defaultLang]
+	cityName := data.City.Names.English
 	metadata.SetValueFunc(ctx, pluginName+"/city/name", func() string {
 		return cityName
 	})
-	countryName := data.Country.Names[defaultLang]
+	countryName := data.Country.Names.English
 	metadata.SetValueFunc(ctx, pluginName+"/country/name", func() string {
 		return countryName
 	})
-	continentName := data.Continent.Names[defaultLang]
+	continentName := data.Continent.Names.English
```

Comment:
Eventually I would like to provide users with the option to set their preferred language in the `geoip` plugin settings, while keeping English as default.

To be clear, no change required in this PR, I will be creating a PR to implement this soon, hopefully we can pack all `geoip` plugin changes together for next release.